test/parallel/test-url.js

```diff
@@ -931,11 +932,13 @@ var parseTestsWithQueryString = {
 for (var u in parseTestsWithQueryString) {
   var actual = url.parse(u, true);
   var expected = parseTestsWithQueryString[u];
-  for (var i in actual) {
-    if (actual[i] === null && expected[i] === undefined) {
+
+  Object.keys(actual).forEach(function (i) {
+    if (i.charAt(0) === "_")
+      expected[i] = actual[i];
+    else if (expected[i] === undefined && actual[i] === null)
       expected[i] = null;
-    }
-  }
+  });
```

Comment:
This effectively ignores any property differences if they start with underscore (private properties, which there were none before), it doesn't change the tests